### PR TITLE
Kuryr migration

### PR DESCRIPTION
**Pull request by gryf**
Attempt to bring automatic way for migrate from Kuryr to OVN-Kubernetes.

### CODE REVIEW

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
from the logs:
```
2023-06-07 14:42:44	node:
```
```json
{'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfigPool', 'metadata': {'creationTimestamp': '2023-06-07T13:18:09Z', 'generation': 4, 'labels': {'machineconfiguration.openshift.io/mco-built-in': '', 'operator.machineconfiguration.openshift.io/required-for-upgrade': '', 'pools.operator.machineconfiguration.openshift.io/master': ''}, 'name': 'master', 'resourceVersion': '125386', 'uid': '224fce4e-702f-4a17-b242-86aa6317cac2'}, 'spec': {'configuration': {'name': 'rendered-master-40900799d27f43b62bf391c96402b4da', 'source': [{'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '00-master'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '01-master-container-runtime'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '01-master-kubelet'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '99-master-generated-registries'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '99-master-ssh'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': 'etcd-on-ramfs'}]}, 'machineConfigSelector': {'matchLabels': {'machineconfiguration.openshift.io/role': 'master'}}, 'nodeSelector': {'matchLabels': {'node-role.kubernetes.io/master': ''}}, 'paused': False}, 'status': {'conditions': [{'lastTransitionTime': '2023-06-07T13:18:37Z', 'message': '', 'reason': '', 'status': 'False', 'type': 'RenderDegraded'}, {'lastTransitionTime': '2023-06-07T13:36:24Z', 'message': '', 'reason': '', 'status': 'False', 'type': 'Updated'}, {'lastTransitionTime': '2023-06-07T13:36:24Z', 'message': 'All nodes are updating to rendered-master-40900799d27f43b62bf391c96402b4da', 'reason': '', 'status': 'True', 'type': 'Updating'}, {'lastTransitionTime': '2023-06-07T14:42:37Z', 'message': 'Node f6zh3sgr-153e1-7lrd7-master-2 is reporting: "failed to drain node: f6zh3sgr-153e1-7lrd7-master-2 after 1 hour. Please see machine-config-controller logs for more information"', 'reason': '1 nodes are reporting degraded status on sync', 'status': 'True', 'type': 'NodeDegraded'}, {'lastTransitionTime': '2023-06-07T14:42:37Z', 'message': '', 'reason': '', 'status': 'True', 'type': 'Degraded'}], 'configuration': {'name': 'rendered-master-146c97cbd5c5117b4ce13416df0e26f3', 'source': [{'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '00-master'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '01-master-container-runtime'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '01-master-kubelet'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '99-master-generated-registries'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': '99-master-ssh'}, {'apiVersion': 'machineconfiguration.openshift.io/v1', 'kind': 'MachineConfig', 'name': 'etcd-on-ramfs'}]}, 'degradedMachineCount': 1, 'machineCount': 3, 'observedGeneration': 4, 'readyMachineCount': 1, 'unavailableMachineCount': 1, 'updatedMachineCount': 1}}
```
and machine-config-controller logs:
```
2023-06-07T14:42:42.994690165Z I0607 14:42:42.994644       1 render_controller.go:569] BaseOSContainerImage=registry.ci.openshift.org/ocp/4.14-2023-06-06-162645@sha256:ddabe5f03cc003a4c6c1a4223fd26009267b444af1ae7b67d6a8f27f7b0f6a4e
2023-06-07T14:42:42.994810947Z I0607 14:42:42.994773       1 status.go:107] Degraded Machine: f6zh3sgr-153e1-7lrd7-master-2 and Degraded Reason: failed to drain node: f6zh3sgr-153e1-7lrd7-master-2 after 1 hour. Please see machine-config-controller logs for more information
2023-06-07T14:42:44.209303024Z I0607 14:42:44.209238       1 drain_controller.go:142] evicting pod openshift-etcd/etcd-guard-f6zh3sgr-153e1-7lrd7-master-2
2023-06-07T14:42:44.217214925Z E0607 14:42:44.217178       1 drain_controller.go:142] error when evicting pods/"etcd-guard-f6zh3sgr-153e1-7lrd7-master-2" -n "openshift-etcd" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
2023-06-07T14:42:49.218185539Z I0607 14:42:49.218108       1 drain_controller.go:142] evicting pod openshift-etcd/etcd-guard-f6zh3sgr-153e1-7lrd7-master-2
```

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

File: ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  steps:
+    pre:
+      - chain: ipi-openstack-pre-noramfs
+    test:
+      - ref: openstack-kuryr-migration
+      - chain: openstack-test-e2e
+    post:
+      - chain: ipi-openstack-post
+    env:
+      CONFIG_TYPE: "minimal"
```
Comment (mandre) [inline ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml:12]:
```suggestion
      CONFIG_TYPE: "proxy"
```

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by dulek** [comment]
Let's note a successful run of the job here. This needs to be rebased once Roman is back.

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by dulek** [comment]
```
error: unable to read image registry.build02.ci.openshift.org/ci-op-6dp33xf2/release@sha256:a0ea86086adc27b2fcc190dea6d068f00c73c47a21bb7b049ed6107b60c393d8: Get "https://registry.build02.ci.openshift.org/v2/": context deadline exceeded
environment: line 139: ret: unbound variable
```
Doesn't seem related.


/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
and again:
```
error: unable to read image registry.build05.ci.openshift.org/ci-op-t4knwfbl/release@sha256:a0ea86086adc27b2fcc190dea6d068f00c73c47a21bb7b049ed6107b60c393d8: Get "https://registry.build05.ci.openshift.org/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)```

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/retest-required
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

File: ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml [hunk 2]
```diff
@@ -13,13 +13,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-shiftstack-shiftstack-ci-main-cleanup-kuryr
-  reporter_config:
```
Comment (EmilienM) [inline ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml:16]: I'm not sure why we're removing the reporting here but we don't want that.

File: ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml [hunk 3]
```diff
@@ -7250,7 +6609,82 @@ periodics:
...
+    repo: shiftstack-ci
+  labels:
+    ci-operator.openshift.io/cloud: openstack-vexxhost
+    ci-operator.openshift.io/cloud-cluster-profile: openstack-vexxhost
+    ci-operator.openshift.io/variant: periodic-4.14
+    ci.openshift.io/generator: prowgen
+    job-release: "4.14"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  minimum_interval: 72h
+  name: periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-e2e-openstack-sdn
```
Comment (EmilienM) [inline ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml:6636]: we want reporting here

File: ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml (hunk 2 above)
Comment (gryf) [reply ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml]: I'm not sure either. Must be some weirdness in updating jobs after rebase. Re-added all  the reports back, and add it to the migration job.

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
(migration has succeeded, some tests has failed)

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

File: ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml (hunk 1 above)
Comment (EmilienM) [reply ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml:12]: @mandre why do you want it to be "proxy"? I think "minimal" is ok.

**Comment by EmilienM** [comment]
/pj-rehearse ack
/lgtm

I think this is fine, if @mandre wants to switch the config type to proxy we can still do it on a separate PR.

Nice work!

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

File: ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml (hunk 1 above)
Comment (mandre) [reply ci-operator/step-registry/openshift/e2e/openstack/kuryr-migration/openshift-e2e-openstack-kuryr-migration-workflow.yaml:12]: I don't think we need the job to be proxy, unless the environment requires it (the above comment is quite old, and the patch was quite different at the time).

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Comment by gryf** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration

**Review by mandre — COMMENTED** [review]
It seems to [have been successful](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/39321/rehearse-39321-periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-openstack-kuryr-migration/1706608570725830656/artifacts/openstack-kuryr-migration/openstack-kuryr-migration/build-log.txt).

Let's merge this!
/lgtm
/hold cancel

**Pull request merged**